pkg/schemas/gen/main/main.go

```diff
@@ -0,0 +1,96 @@
+//go:build exclude
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"go/format"
+	"log"
+	"os"
+	"path/filepath"
+	"slices"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/schemas/gen"
+	"golang.org/x/exp/maps"
+)
+
+func main() {
+	file := os.Getenv("GOFILE")
+	fmt.Printf("Running generator on %s with args %#v\n", file, os.Args[1:])
+
+	allStructsDetails := make([]gen.Struct, len(gen.SdkShowResultStructs))
+	for idx, s := range gen.SdkShowResultStructs {
+		allStructsDetails[idx] = gen.ExtractStructDetails(s)
+	}
+
+	printAllStructsFields(allStructsDetails)
+	printUniqueTypes(allStructsDetails)
+	generateAllStructsToStdOut(allStructsDetails)
```

Comment:
I think these prints should be enabled by a flag or env.

Reply:
Please check the README, this point is there.